folder_structure.md

```diff
@@ -254,6 +255,28 @@ The files are all produced by the [profiling-recipe](https://github.com/jump-cel
 
 - `2021_04_26_Batch1` is the batch and `BR00117035` is the plate
 
+### `embeddings` folder structure:
+
+The structure here is very similar to `analysis`
+
+```
+└── embeddings
+    ├── 2021_04_26_Batch1
+    │   ├── BR00117035
+    │   │   └── efficientnet_v2_imagenet1k_s_feature_vector_2_ec756ff
+    │   │       ├── BR00117035-A01-1
+    │   │       │   └── embedding.parquet
+    │   │       └── BR00117035-A01-2
+    │   └── BR00117036
+    └── 2021_05_31_Batch2
+```
+
+In this example batch:
+- `2021_04_26_Batch1` is the batch and `BR00117035` is the plate
+- `efficientnet_v2_imagenet1k_s_feature_vector_2_ec756ff` is an identifier for the deep learning network, prefixed with some hash for the model
```

Comment:
Is the hash a necessary part of the name?

Reply:
The goal here is to come up with a unique ID for the model. I'd be just as happy with a DOI (10.5281/zenodo.7114558 for CellPainting CNN v1)

Reply:
Sounds good!